src/components/widgets-common/chart-threshold/chart-threshold.component.coffee

```diff
@@ -0,0 +1,109 @@
+###
+#   @desc Chart Threshold - create Widget KPIs from Widget charts. (Highchart widgets only).
```

Comment:
No but it complies to JSdoc, so there may be tools to do so. As it is, I just like the way github / sublime colorise the tags for easier reading.